classytags/utils.py

```diff
@@ -83,8 +83,9 @@ def mixin(parent, child, attrs={}):
 def flatten_context(context):
     if callable(getattr(context, 'flatten', None)):
         return context.flatten()
-    else:
+    elif hasattr(context, 'dicts'):
```

Comment:
Thanks for your explanation. I hope I could do something useful for this
interesting project.

On Mon, Jan 4, 2016 at 6:10 AM Jonas Obrist <notifications@github.com>
wrote:

> In classytags/utils.py
> <https://github.com/ojii/django-classy-tags/pull/29#discussion_r48701531>:
>
> > @@ -83,8 +83,9 @@ def mixin(parent, child, attrs={}):
> >  def flatten_context(context):
> >      if callable(getattr(context, 'flatten', None)):
> >          return context.flatten()
> > -    else:
> > +    elif hasattr(context, 'dicts'):
>
> I'm not convinced that this is the proper way to do this, though I haven't
> fully looked into it. Maybe it should instead be elif isinstance(context,
> Context) (or isinstance(context, dict)?). I also noticed there's no test
> for this function directly, which is my fault and I'll try to fix this
> possibly tonight when I get off work. But I think I can take it from here,
> so you don't have to worry about this too much.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ojii/django-classy-tags/pull/29/files#r48701531>.
>